### PR TITLE
Add cron scheduler and fix stock quantity calculation issue

### PR DESCRIPTION
**Pull request by BushraAsif**
- Fix: Stock quantity calculation issue
- Add cron scheduler to change the status of the pending order to cancel

### CODE REVIEW

**Review by shahbaz-mehar — COMMENTED** [review]
Changes have been reviewed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.3.0]
+
+**Improvements**
+Add cron scheduler to change the status of pending order to canceled
```
Comment (emicha) [inline CHANGELOG.md:7]: If this is based on a configuration value, please state so.

File: Model/Config/CronConfig.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+         parent::__construct($context, $registry, $scopeConfig, $cacheTypeList, $resource, $resourceCollection, $data);
+     }
+
+     /**
+      * @return mixed
+      * @throws \Exception
+      */
+     public function afterSave()
+     {
+        $time = $this->getData('groups/sdm_altapay_config/groups/cronScheduled/fields/time/value');
```
Comment (emicha) [inline Model/Config/CronConfig.php:77]: Please update the documentation explaining how to setup this cronjob.

File: CHANGELOG.md (hunk 1 above)
Comment (BushraAsif) [reply CHANGELOG.md]: This has been updated

**Review by emicha — APPROVED** [review]

**Pull request merged**